script.js

```diff
@@ -1,99 +1,12 @@
-/* add new book */
+import { newBook } from './modules/add.js';
+import { remove } from './modules/remove.js';
+import { show } from './modules/show.js';
```

Comment:
As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/201755311-a5009e79-6adf-4112-b3b4-68d2cb19ac43.png)

Your project should display the current date, below the navigation bar 😉.
Tip: to do so, you can follow the [luxon installation guide](https://moment.github.io/luxon/#/install?id=es6) and import it as a module, just like you did with your own modules 😄.

The objective of this part is to assure that you understand how to handle libraries, as they are extremely common and useful.

Reply:
well resolved! ✔️